hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -120,19 +120,8 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     final long requestedSize = keyArgs.getDataSize() > 0 ?
         keyArgs.getDataSize() : scmBlockSize;
 
-    boolean useRatis = ozoneManager.shouldUseRatis();
```

Comment:
I agree to remove these unnecessary checks. They should not be null.

Reply:
keyArgs.getFactor() and keyArgs.getType() have `@NotNull` annotation on return value.